mautrix_telegram/license.py

```diff
@@ -0,0 +1,46 @@
+# mautrix-telegram - A Matrix-Telegram puppeting bridge
+# Copyright (C) 2021 Tulir Asokan
+# Copyright (C) 2022 New Vector Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+from uuid import uuid4
+import os
+
+from mautrix.util.logging import TraceLogger
+
+_LICENCE_FILE_PATH = os.environ.get(
+    "MAUTRIX_TELEGRAM_LICENCE_PATH", os.path.abspath("../instanceId")
```

Comment:
To avoid future frustration: "licen**c**e" or "licen**s**e"?  (If it were only in log messages, it'd be fine, but misspelled config variables can easily lead to hair-pulling experiences :slightly_smiling_face: )